tests-integ/test_nova_reels.py

```diff
@@ -0,0 +1,97 @@
+import os
+import re
+import time
+import uuid
+from unittest.mock import patch
+
+import boto3
+import pytest
+from strands import Agent
+from strands_tools import nova_reels
+
+AWS_REGION = "us-east-1"
+TEST_BUCKET = f"nova-reels-e2e-test-{str(uuid.uuid4())[:8]}".lower()
+INVOKE_ARN = f"arn:aws:bedrock:{AWS_REGION}:123456789012:async-inference/{uuid.uuid4()}"
```

Comment:
Is this needed at all?  This is specific to an account which concerns me a little